src/modules/form/tableSettingsForm.tsx

```diff
@@ -0,0 +1,199 @@
+import { Formik, Form, Field } from 'formik';
+import { z } from 'zod';
+import { Button } from '@/components/button';
+import { Input } from '@/components/input';
+import { Typography } from '@/components/typography';
+import { Select } from '@/components/select';
+import { Checkbox } from '@/components/checkbox';
+
+const TableSchema = z.object({
```

Comment:
optionally, can be moved to a separate file to keep file simple and follow single responsibility